xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/ModelFactory.java

```diff
@@ -1063,4 +1076,40 @@ public JobLog toRestJobLog(Collection<LogEvent> logs, URI self)
 
         return log;
     }
+
+    /**
+     * check if the given property should be exposed via REST
+     * @param restProperty the property to be read/written
+     * @param forWrite true iff we want to write to the XObject property
```

Comment:
s/iff/if

Reply:
I'm not sure I understand the `forWrite` parameter.

Reply:
``forWrite`` : do we want to read(false) or write(write) the property. Used e.g. to still allow to write the email via PUT (I can think of use cases for that). I first named it ``isWrite`` but was not sure if ``forWrite`` is better. Maybe use ``writeAccess`` instead?
the ``iff`` is a shortcut for ``if and only if`` ... ok, seems to be an unknown abbreviation.